scripts/mesh-point/mesh.adhoc

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -e
+
+# Set wireless regulatory domain
+sudo iw reg set CA
+
+# Select physical device that supports 802.11s Mesh Point
+
+mesh_dev="wlan1"
```

Comment:
Currently manually specified.  
Should probably be wlan0 since now the  wlan0 (onboard) will not by renamed to wlan-ap on boot.

Any thoughts on how to automate this?